index.html

```diff
@@ -3055,10 +3055,21 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="dialog">
 			<rdef>dialog</rdef>
 			<div class="role-description">
-				<p>A dialog is a descendant window of the primary window of a web application. For HTML pages, the primary application window is the entire web document, i.e., the <code>body</code> element.</p>
-				<p>Dialogs are most often used to prompt the user to enter or respond to information. A dialog that is designed to interrupt workflow is usually modal. See related <rref>alertdialog</rref>.</p>
+				<p>A dialog is a descendant window of the primary window of a web application. For HTML pages, the primary application window is the entire web document, i.e., the content the user interacts with, within the <code>body</code> element.</p>
```

Comment:
note for reviewers:
do we want to continue to say the `body` element here?   it's not technically the 'entire web document' but is as far as the user likely cares about - which is what prompted my edit to this line.   open to other suggestions of how this could be worded.